python/lsst/pex/config/config.py

```diff
@@ -1384,9 +1365,12 @@ def saveToStream(self, outfile, root="config", skipImports=False):
                 configType = type(self)
                 typeString = _typeStr(configType)
                 outfile.write(f"import {configType.__module__}\n")
+                # We are required to write this on a single line because
```

Comment:
As a general comment, I have seen people reformat config .py files to make them look neater and I now worry that those will no longer be readable.